src/Topology.jl

```diff
@@ -131,3 +131,26 @@ function remove_bond!(topology::Topology, i::Integer, j::Integer)
     )
     return nothing
 end
+
+function add_residue!(topology::Topology, residue::Residue)
+    check(
+        lib.chfl_topology_add_residue(topology.handle, residue.handle)
+    )
+    return nothing
+end
+
+function residues(topology::Topology)
```

Comment:
This function returns the number of residues, so calling it `residues` feels strange. How about `residues_count`? I don't know which similar pattern is used in Julia Base.